mobile/src/main/res/values/strings.xml

```diff
@@ -183,4 +183,6 @@
     <string name="content_description_open_number_picker">Open number picker</string>
     <string name="content_description_video">Video</string>
     <string name="content_description_open_openhab">Open openHAB</string>
+    <string name="swipe_to_refresh_description">The app refreshes widgets regularly, you mostly do not need to do it yourself.</string>
```

Comment:
Maybe describe Kai's usecase from https://github.com/openhab/openhab-android/issues/539#issuecomment-355902286

Reply:
The problem is, that the snackbar message is very limited, or it will be truncated (this can maybe "turned off"?), so it needs to be somehow short. This is the one I came up with, which is short enough but still descriptive :]

Reply:
I think its better to remove the full stop in this sentence, otherwise lgtm

Reply:
I'd make it say `The app also refreshes automatically, you normally do not need to do it manually`